main.js

```diff
@@ -0,0 +1,142 @@
+// The app collection
+const AppCollection = Backbone.Collection.extend({
```

Comment:
This is an incorrect use of a collection - it should be full of some kind of model. You need at least a video model.

It is missing the `model` attribute.